### PR TITLE
Class balancing

### PR DESCRIPTION
**Pull request by GabrielVrabie007**
Traning and balance a classification model 
@danielavornic

### CODE REVIEW

**Review by danielavornic — CHANGES_REQUESTED** [review]
Nice work!

Some things I have to mention:
1. At comment `# Drop unusefull column`: it needs to be `df = df.drop(columns=["#"])
` to modify the dataframe, not just `df.drop(...)`
2. It's best to split the data into train/test datasets first, and then balance only the training part. Otherwise, the test set ends up with fake samples that were made using informatin from the training data, so your results look better than they really are. Here it's also worth mentioning to train the model for every trained dataset.

Other than that, the evaluation of the models is also great :>

**Comment by GabrielVrabie007** [comment]
Hi Daniela!
At the moment,I am a little busy at work and also have some problems with my personal laptop.I will change it and push it next week
